From e4c7b7ace652d40df1de7d078e21af6b15bd9c3d Mon Sep 17 00:00:00 2001
Message-Id: <e4c7b7ace652d40df1de7d078e21af6b15bd9c3d.1413837857.git.chang-joon.lee@intel.com>
In-Reply-To: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
References: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
From: Deepak S <deepak.s@intel.com>
Date: Sat, 6 Sep 2014 18:01:07 +0530
Subject: [PATCH 270/312] FOR_UPSTREAM [VPG]: Add rc6 debugfs support to vlv &
 chv.

Looks like the vlv debugfs support patch was missed with
latest forklift

Issue: GMIN-3469
Change-Id: Ie343d869e5af5e213b7aca205bd97f5b0a8e73dd
Signed-off-by: Deepak S <deepak.s@intel.com>
---
 drivers/gpu/drm/i915/i915_debugfs.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_debugfs.c b/drivers/gpu/drm/i915/i915_debugfs.c
index 2ddab13..8965c6d 100644
--- a/drivers/gpu/drm/i915/i915_debugfs.c
+++ b/drivers/gpu/drm/i915/i915_debugfs.c
@@ -4690,7 +4690,7 @@ static int i915_rc6_disable_get(void *data, u64 *val)
 	struct drm_device *dev = data;
 	struct drm_i915_private *dev_priv = dev->dev_private;
 
-	if ((INTEL_INFO(dev)->gen < 6) || IS_VALLEYVIEW(dev))
+	if ((INTEL_INFO(dev)->gen < 6))
 		return -ENODEV;
 
 	flush_delayed_work(&dev_priv->rps.delayed_resume_work);
@@ -4706,7 +4706,7 @@ static int i915_rc6_disable_set(void *data, u64 val)
 	struct drm_i915_private *dev_priv = dev->dev_private;
 	int ret;
 
-	if ((INTEL_INFO(dev)->gen < 6) || IS_VALLEYVIEW(dev))
+	if ((INTEL_INFO(dev)->gen < 6))
 		return -ENODEV;
 
 	flush_delayed_work(&dev_priv->rps.delayed_resume_work);
-- 
1.7.9.5

